packages/components/app/styles/components/tag.scss

```diff
@@ -0,0 +1,101 @@
+//
+// TAG COMPONENT
+//
+// properties within each class are sorted alphabetically
+//
+//
+
+@use "../mixins/focus-ring" as *;
+
+$hds-tag-border-radius: 13px;
+
+.hds-tag {
+  align-items: stretch;
+  background-color: var(--token-color-surface-interactive);
+  border: 1px solid var(--token-color-border-strong);
+  border-radius: $hds-tag-border-radius;
+  color: var(--token-color-foreground-primary);
```

Comment:
```suggestion
  color: var(--token-color-surface-interactive);
```

Reply:
@agendelHC `color` refers to the text or icon (foreground) color, not the background color of the component, so we wouldn't use `surface` colors for it. According to the [designs](https://www.figma.com/file/noyY6dUMDYjmySpHcMjhkN/branch/LdUGD1ub9oPL28gv5w4Tdz/HDS-Product---Components?node-id=13720%3A34360), this token is actually correct. Does something look off to you?